### PR TITLE
Oxygen heals slight toxin damage for humans

### PR DESCRIPTION
**Pull request by Irradiation**
>Vox
If they slurp the gas that they need to breath, they get healed.

>Humans
If they slurp O2, nothing happens.

PS: Plasma healing Plasmemes PR soon maybe, because plasma is what's actively holding their body together and such. Also plasma bar drinks.

[why][consistency][honk][tested]

:cl:
 * tweak: Ingesting Oxygen now heals slight toxin damage for humans.

### CODE REVIEW

**Comment by Kurfursten** [comment]
Seems fine but I'd rather go the opposite way and remove nitrogen healing Vox

HOT OPINIONS

**Comment by Irradiation** [comment]
Now there's a thought.

**Comment by Irradiation** [comment]
Nether does Nitrogen healing oxygen & toxin damage in Birbs tbh

**Comment by jknpj** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1808645/

**Comment by Irradiation** [comment]
tl;dr?

**Comment by Irradiation** [comment]
I got my tag valids. Time to go home.

**Comment by Irradiation** [comment]
wrong button

**Comment by SonixApache** [comment]
Nitrogen heals vox tox because vox breathing oxygen causes vox tox
The opposite is not true for humans and nitrogen.
